main.css

```diff
@@ -620,17 +620,23 @@ body {
 	   /* Add shadows to create the "card" effect */
 	   box-shadow: 0 4px 8px 0 rgba(0,0,0,.1);
 	   background-color: #ddd;
-	   transition: box-shadow 0.1s ease-in-out;
-	   
+	   transition: box-shadow 0.1s ease-in-out;   
    }
+
    /* On mouse-over, add a deeper shadow */
    .column:hover {
 	   box-shadow: 0 8px 16px 0 rgba(0,0,0,.3);
+	   border: 

```

Comment:
empty?

Reply:
That's weird; it's a remnant from experiementing and I committed db522b4 to undo it. Not sure why it showed up for you to review.